targets/ath79-generic

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
@blocktrron Do we need to (a) implement ct/non-ct switching here? And if not, do we need to (b) force non-ct here?

Reply:
@mweinelt It is not necessary for 11s to work (as the QCA9886 is a W2 chip), however non-ct has a more recent firmware base which is why we use it for most targets.

So yes, the Firmware selection should be done like it is done for ar71xx / ipq40xx.

Reply:
I removed this after the linter complained about unused variables.